SchoolNormalization.md

```diff
@@ -0,0 +1,235 @@
+SchoolNormalizationService
+=============
+
+Table of Contents
+_________
+- [Request Information](#request-information)
+- [Sample Response](#sample-response)
+- [Versioning](#versioning)
+- [Supported Countries](#supported-countries)
+
+
+
+#Request Information
+
+
+HTTP method: GET or POST
+Parameters (query/form):
+-        query (required) : The educational instutution information to be queried.
+-        country (optional) : 2 digit ISO country code of eductaionl institution. Accepted country codes are [here](#supported-countries)
+ 
+Example: https://api.careerbuilder.com/core/normalizedschools?query=georgia%20tech&country=us
+
+#Sample Response
+
+
+```
+{
+    "data": {
+        "normalized_schools": [
+            {
+                "normalized_school_name": "Georgia Institute of Technology",
+                "id": "53bff579e4b04710d09fa98d",
+                "country": "US",
+                "surface_form": "georgia tech",
+                "confidence": 2
+            }
+        ]
+    }
+}
+```
+
+#Versioning
+-----------
+The response from the School Normalization Service is versioned with the current version being 1.0. The data set used to preform the normalization is also versioned with the current version being 1.0.0, however customers will not have the ability to use older versions of the data set as the updates are done to improve the accuracy of the results rather, and will not impact the response.
```

Comment:
I would reword the second sentence here, I think it's a little confusing. Maybe something like...

"The data set used to perform the normalization is also versioned (currently 1.0.0). This data set is upgraded at will in order to improve accuracy of the classifier. Requests made to this service will always use the latest version of the data set."